terraform/development/main.tf

```diff
@@ -33,12 +33,39 @@ locals {
 module "database" {
   source = "../modules/rds"
 
-  prefix                            = local.prefix
-  allocated_storage                 = 5
-  db_subnet_group_name              = module.networking.db_private_subnet_group_name
-  ingress_source_security_group_ids = []
-  instance_class                    = "db.t3.micro"
-  vpc_id                            = module.networking.vpc_id
+  prefix                             = local.prefix
+  allocated_storage                  = 5
+  db_subnet_group_name               = module.networking.db_private_subnet_group_name
+  ingress_from_ecs_security_group_id = module.ecs.ecs_security_group_id
+  instance_class                     = "db.t3.micro"
+  vpc_id                             = module.networking.vpc_id
+}
+
+module "ecs" {
+  source = "../modules/ecs"
+
+  prefix = local.prefix
+  additional_task_role_policy_arns = {
+    "RDS_access" : module.database.rds_data_access_policy_arn
+    "Redis_access" : "arn:aws:iam::aws:policy/AmazonElastiCacheFullAccess"

```

Comment:
It feels like we might not be able to control who can read/write to Redis using an IAM policy (for example none of the elasticache actions listed in AWS console seem to be what we'd configure - see image)

It looks like this might need to be configured in Redis itself using an ACL ([see here](https://redis.io/docs/management/security/acl/) for example)

For now I've left this in, and down the line we'll need to check if it's actually needed to use Redis with the app

![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data-infrastructure/assets/109098469/3132c3a9-de63-49ed-a131-1f3b386875a8)